app/views/course_templates/index.html.erb

```diff
@@ -19,11 +19,11 @@
         <td><%= course_template.name %></td>
         <td><%= course_template.title %></td>
         <td><%= course_template.description %></td>
-        <td><%= course_template.material_url %></td>
-        <td><%= course_template.source_url %></td>
+        <td><%= link_to course_template.material_url, course_template.material_url %></td>
+        <td><%= link_to course_template.source_url, course_template.source_url %></td>
```

Comment:
I also see people accidentally clicking the link while attempting to copy it.
Though this information is rarely used anyways.